bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -61,6 +61,11 @@ <h2 class="heading">
           <a href="{{ url('firefox.sms') }}">Text a download link to my Android phone</a>
         </p>
       </div>
+
+      {% if l10n_has_tag('ios_announce') %}
+        <p class="ios-link">{{_('Firefox is coming soon to iOS! <a href="{url}">Sign up to learn more</a>').format(url=url('newsletter.ios'))|safe }}</p>
```

Comment:
Can you add a period after the link? That would let us reuse the same string in the 3 files.